### PR TITLE
Adds detection for BYJU'S, CX, ELE-GATE, G-PLUS, Humanware, KENWOOD, LongTV, Mode Mobile, M-KOPA, Okapi, RCT, Realix, Trimble brands and improves detection for existing ones

### PR DESCRIPTION
**Pull request by liviuconcioiu**

### CODE REVIEW

File: Tests/fixtures/smartphone-18.yml [hunk 1]
```diff
@@ -3273,3 +3273,327 @@
...
+    version: 8.1.0
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 99.0.4844.73
+    engine: Blink
+    engine_version: 99.0.4844.73
+  device:
+    type: smartphone
```
Comment (sanchezzzhak) [inline Tests/fixtures/smartphone-18.yml:3469]: tablet https://shop.cashconverters.co.nz/Listing/Details/61595821/Everis-Tablet-32Gb-E0111

File: Tests/fixtures/smartphone-18.yml [hunk 2]
```diff
@@ -3273,3 +3273,327 @@
...
+    version: "9"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 107.0.0.0
+    engine: Blink
+    engine_version: 107.0.0.0
+  device:
+    type: smartphone
```
Comment (sanchezzzhak) [inline Tests/fixtures/smartphone-18.yml:3487]: tablet

File: Tests/fixtures/smartphone-18.yml [hunk 3]
```diff
@@ -3273,3 +3273,327 @@
...
+    version: "9"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 110.0.0.0
+    engine: Blink
+    engine_version: 110.0.0.0
+  device:
+    type: smartphone
```
Comment (sanchezzzhak) [inline Tests/fixtures/smartphone-18.yml:3505]: tablet https://www.productreview.com.au/listings/everis-android-90-tablet-e0114

Comment (liviuconcioiu) [reply Tests/fixtures/smartphone-18.yml]: All Everis devices were tablets. I've fixed the other too.

**Review by sanchezzzhak — APPROVED** [review]

**Pull request merged**